views/cypress/tests/item-authoring-response-processing.spec.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA ;
+ */
+
+import urls from '../utils/urls';
+import selectors from '../utils/selectors';
+
+import { addInteraction } from '../utils/authoring-add-interactions';
+import { addResponseProcessing } from '../utils/set-response';
+
+describe('Item Authoring', () => {
+    const className = 'Test E2E class';
+    const itemName = 'Test E2E item 1';
+
+    const commonWidgetSelector = (qtiClass) => `.widget-box.widget-blockInteraction[data-qti-class="${qtiClass}"]`;
+
+    const interactions = {
```

Comment:
no need all array, you can just use 1 variable

`const choice = 'choiceInteraction',`

Reply:
yes,  added them because I was considering to add more interactions but then never did. will remove, thanks!